ci_support/run_docker_build.sh

```diff
@@ -37,6 +37,8 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
+# library dependencies for matplotlib
+yum install -y libXext libXrender libSM tk libX11-devel
```

Comment:
I'm afraid these will disappear when the conda-smithy re-renders. Some of these are reasonable to expect of the OS (though I'm not sure about tk) and can be pushed up into the docker image.

Essentially, I think this is a temporary solution (which I'm OK with) that we need to find longer term solutions to.